src/Examples.Console/ExampleDynatraceExporter.cs

```diff
@@ -27,26 +27,33 @@
 
 namespace Examples.Console
 {
-    internal class TestDynatraceExporter
+    internal class ExampleDynatraceExporter
```

Comment:
```suggestion
    internal class DynatraceExporterExample
```
Otherwise it sounds like an exemplary implementation of a DynatraceExporter.